matching-notification-resources/src/main/resources/resources/uicomponents/matchingNotification/matchesTable.js

```diff
@@ -210,11 +209,8 @@ var PhenoTips = (function (PhenoTips) {
                 disclaimerContainer.insert(new Element('div', {'class' : 'server-name'}).insert(serverName));
 
                 var endTimeInput = trigger.up('label').down('.endTime');
```

Comment:
Now that the nature of the input value has changed from date to message, the variable name should change too so that the code is easier to understand. What about something like "lastMatchRunInfo"? The same thing goes for the CSS class of the input.

Reply:
Where do you think is better place for message generation - velocity or javascript?

Reply:
I think either is fine. Generating this particular message on the backend does make it easier to handle translations properly. In general, the code is easier to understand and to maintain if the functionality is not split between frontend and backend, so you could even go one step further and generate the whole content of the tooltip on the backend before putting it in a hidden input or in a `data-` attribute. Just my 2 cents.

Reply:
On the other side last run time value is used for 2 places, both in disclaimers and in dropdown for servers. Messages differ, so better shift their generation to js.